docs/en/loading/automq-routine-load.md

```diff
@@ -18,7 +18,7 @@ For an understanding of the basic principles of Routine Load, refer to the secti
 
 Ensure you have a running StarRocks cluster. For demonstration purposes, this article follow the [deployment guide](../quick_start/deploy_with_docker.md) to install a StarRocks cluster on a Linux machine via Docker.
 
-Creating a database and test table with the primary key model:
+Creating a database and test table with the primary key table:
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/176a9d7e-9057-49aa-954a-0453c6da6784)
two "tables". I think you mean "a Primary Key table for testing". See to this in all docs involved.